exercises/G-numbers/exercise.js

```diff
@@ -1 +1,5 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15;
+var numberOfMentors = 8;
+let total = numberOfMentors + numberOfStudents;
+console.log('Total number of students and mentors:' + ' ' + total);
```

Comment:
You can add a space at the end of the string here  rather than having  `+ ' '`
`console.log('Total number of students and mentors: '  + total);`